File: programmers/Level1_1_5.py
def solution(s):
    len_s = len(s)
    answer = ''

    if len_s % 2 == 0:
        answer = s[int(int(len_s/2)-1):int(int(len_s/2)+1)]
    else:
        answer = (s[int(len(s)/2)])
    
    return answer

File: programmers/scala/Level1_1_5.scala
// https://programmers.co.kr/learn/courses/30/lessons/12903?language=scala

object Solution {
    def solution(s: String): String = {
        if (s.length() % 2 == 0)
            return s.substring((s.length()/2)-1, s.length() / 2 + 1)
        else
            return s.substring((s.length()-1) / 2, (s.length()+1) / 2)
    }
}